apis/common/v1/resource.go

```diff
@@ -203,6 +203,11 @@ type ResourceSpec struct {
 	// Deprecated: Please use ProviderConfigReference, i.e. `providerConfigRef`
 	ProviderReference *Reference `json:"providerRef,omitempty"`
 
+	// ManagementPolicy specifies how this managed resource should be managed.
```

Comment:
This seems a bit vague, I tried putting it in different words but feel free to discard/change
```suggestion
	// ManagementPolicy specifies the level of control Crossplane has over the managed external resource.  
    // The options are: 
    // FullControl (default) - fully controlled by Crossplane controllers, including deletion.
    // ObserveOnly - only observed by Crossplane controllers, not modified or deleted.
    // OrphanOnDelete - orphaned when its managed resource is deleted.
```